package.json

```diff
@@ -50,35 +50,52 @@
     "@google-cloud/error-reporting": "^2.0.4",
     "@lhci/cli": "^0.7.2",
     "@rollup/plugin-node-resolve": "^8.4.0",
+    "@sanity/block-content-to-markdown": "^0.0.5",
+    "@sanity/client": "^3.3.3",
+    "@sanity/image-url": "^1.0.1",
```

Comment:
It would make me very happy if we could start pinning dependencies, removing the caret and hence allowed version ranges. This makes builds  and dev setups more reproducible which makes operation and support a lot easier, no matter what environment the user is on or package manager used.